tools/static-assets/server/boot.js

```diff
@@ -438,13 +438,17 @@ var loadServerBundles = Profile("Load server bundles", function () {
   });
 });
 
+var fiberStartupHook = "_meteorInsideStartupHook";
```

Comment:
IIRC this should be a `Symbol` so as to not be included when cloned, right? @OleksandrChekhovskyi 
See PerformanceMonitor `perfDataSymbol`

Reply:
Ok, maybe we need it to be a Symbol after all, due to potential `cloneFiberOwnProperties` issues. But then we need a nice place for an accessor, or do a hack by finding Symbol by name.

Reply:
Maybe instead we could define the Symbol and the accessor (`Meteor._something`) in `startup_server.js` (where we could wrap the callback).

Reply:
Fixed by adding `Meteor._isInsideStartupHook`